src/catkin_pkg/templates/package.xml.in

```diff
@@ -41,8 +41,9 @@
 
   <!-- The export tag contains other, unspecified, tags -->
   <export>
-    <!-- You can specify that this package is a metapackage here: -->
-    <!-- <metapackage/> -->
+    <!-- Specify that this is a metapackage using <metapackage/> here -->
+    <!-- Consider using catkin_create_pkg --meta if that is the case -->
```

Comment:
Please just remove the new as well as the old comment here since the tool now covers that configuration. It is not necessary to have the comment in the generated output anymore.